rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/provider/IndexJavaSourceFilesRuleProvider.java

```diff
@@ -76,7 +80,7 @@ private IndexJavaFileIterationOperator()
         @Override
         public void perform(GraphRewrite event, EvaluationContext context, JavaSourceFileModel payload)
         {
-            WindupRoasterWildcardImportResolver.setGraphContext(event.getGraphContext());
+            WindupWildcardImportResolver.setGraphContext(event.getGraphContext());
```

Comment:
It's hacky because GraphContext isn't injectable and this class is sometimes used with Roaster as well (where it needs to be located as a Service). It could support both approaches, but I'm not sure that is better.